plugin-mainmenu/xdgcachedmenu.cpp

```diff
@@ -73,6 +73,12 @@ XdgCachedMenu::XdgCachedMenu(MenuCache* menuCache, QWidget* parent): QMenu(paren
 {
     // qDebug() << "CREATE MENU FROM CACHE" << menuCache;
     MenuCacheDir* dir = menu_cache_get_root_dir(menuCache);
+
+    // get current desktop name or fallback to LXQt
+    const QByteArray xdgDesktop = qgetenv("XDG_CURRENT_DESKTOP");
+    const QByteArray desktop = xdgDesktop.isEmpty() ? "LXQt:X-LXQt:LXQT:X-LXQT" : xdgDesktop;
```

Comment:
isn't this enough ?
``` C++
const QByteArray desktop = xdgDesktop.isEmpty() ? "LXQt" : xdgDesktop;
```
`XDG_CURRENT_DESKTOP` is set to `LXQt` in the `startlxqt` script.